ex/loqui/lib/loqui/client.ex

```diff
@@ -1,5 +1,89 @@
 defmodule Loqui.Client do
+  @moduledoc """
+  # A high performance client for the Loqui protocol
+
+  This client exposes five public functions, `start_link`, `request`, `push`, `ping` and `close`.
```

Comment:
This is if you want to manually ping the server for some reason. I can remove it, it's not essential.